pkg/neofetch

```diff
@@ -6,6 +6,9 @@ filesize=68930
 sha512=0a13acadf1841c1e714c07f9643028446a1f4df6a79e7c3189c4c11d8854813dfbb983784d62dc59a9209893ee2339ca5642fe413d1066a5cb6e9e167ae74975
 tardir=neofetch-3.1.0
 
+[deps]
+bash
+
```

Comment:
in case that's a build-time host-tool dependency, it should go into deps.host , and/or if it is a run-time dependency, deps.run (see python packages for examples). since afaik bash doesn't ship a libbash one could link against at buildtime, deps certainly is not the right thing to use